source/features/ci-link.css

```diff
@@ -3,3 +3,42 @@
 	pointer-events: none;
 	margin-left: 0.3em;
 }
+
+.gh-header-title > .js-socket-channel {
+	display: inline;
+}
+
+/* Ensure the CI details dropdown always opens to the right */
+.gh-header-title .commit-build-statuses .dropdown-menu {
+	left: 100%;
+	right: auto;
+	margin: 24px 0 0 8px;
+}
```

Comment:
Opening the dropdown on the right seems like the safer option, but it will still get clipped if the window is narrow and PR title ends all the way to the right.

Idk if GitHub has "automatic" dropdowns that open on the correct side, I haven't found anything in their style guide.

Reply:
Maybe then it should be positioned elsewhere, not next to the title.